backend/routes/health.py

```
@@ -67,17 +67,17 @@ async def health():
-                xpu_name = torch.xpu.get_device_name(0)
-            except Exception:
-                xpu_name = "Intel GPU"
-    except ImportError:
-        pass
+    # Native XPU support in PyTorch 2.4+; IPEX optional for enhanced performance
+    if hasattr(torch, "xpu") and torch.xpu.is_available():
+        has_xpu = True
+        try:
+            import intel_extension_for_pytorch  # noqa: F401 -- enhances XPU perf if available
+        except ImportError:
+            pass
```

Ruff (0.15.10)

[warning] Use `contextlib.suppress(ImportError)` instead of `try`-`except`-`pass`

(SIM105)